docs/development/releasing.md

```diff
@@ -4,19 +4,19 @@
 1. Version starts with "v", e.g. "vX.Y.Z".
 1. Fill "Raw notes".
     - `git fetch --tags upstream`
-    - `git log --pretty="%h - %s" v0.11.1..HEAD`
+    - `git log --pretty="%h - %s" v0.12.1..HEAD`
```

Comment:
The `git describe --tags --abbrev=0` will yield the last tag, so let's put a more clever script here:

```
git log --pretty="%h - %s" $(git describe --tags --abbrev=0)..HEAD
```

Reply:
Thanks, that works perfectly!